test/timeseries_path_partitioner/processor-spec.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const { TestContext } = require('@terascope/job-components');
+const { DataEntity } = require('@terascope/utils');
+const Processor = require('../../asset/timeseries_path_partitioner/processor');
+
+describe('Timeseries router', () => {
```

Comment:
Since this is a new spec file, this should use the `WorkerHarness` from `teraslice-test-harness`.

Reply:
Updated test to use teraslice-test-harness